wp-content/civi-extensions/goonjcustom/Civi/InductionService.php

```diff
@@ -195,7 +261,7 @@ public static function volunteerInductionAssignee($op, $groupID, $entityID, &$pa
       return;
     }
 
-    if (!$inductionFields['Assign']) {
+    if (!$inductionsFields['Assign']) {
```

Comment:
Update this variable name at all places
```suggestion
    if (!$inductionFields['Assign']) {
```

Reply:
sure